assets/js/modules/sign-in-with-google/components/common/AnyoneCanRegisterDisabledNotice.js

```diff
@@ -97,12 +97,12 @@ export default function AnyoneCanRegisterDisabledNotice( { className } ) {
 				sprintf(
 					/* translators: %1$s: Setting name, %2$s: Sign in with Google service name */
 					__(
-						'Enable the %1$s setting to allow your visitors to create an account using the %2$s button. <br/>Visit <a>WP settings</a> page to manage this membership setting.',
+						'Enable the %1$s setting to allow your visitors to create an account using the %2$s button. <br/>Visit <a>WordPress Settings</a> to manage this setting.',
```

Comment:
I agree with the proposed changes here, but the current text does match what we have in the design. See https://www.figma.com/design/h9yGR15Pg2DUtgERW4AFAv/Sign-in-with-Google?node-id=1028-2947